src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -149,6 +164,8 @@ export class SkySelectFieldComponent implements ControlValueAccessor {
   }
 
   public openPicker() {
+    this.isModalOpen = true;
```

Comment:
Think we could move this statement to just beneath the `this.modalService.open` call, just in case the `resourcesService.getString` call is interrupted somehow?

```
const modalInstance = this.modalService.open(SkySelectFieldPickerComponent, {
  providers: [{
    provide: SkySelectFieldPickerContext,
    useValue: pickerContext
  }]
});

this.isModalOpen = true;
```

Reply:
Alright, I was worried about the modal opening stealing focus too fast, but it looks like that is consistently not happening. So I've moved it down